README.md

```diff
@@ -389,14 +389,17 @@ If you are using `babel-plugin` you might not need to apply `webpack-loader` to
 
 ### Code Splitting
 
-If you want to use Code Splitting + React Hot Loader, the simplest solution is to pick one of our compatible library:
+If you want to use Code Splitting + React Hot Loader, the simplest solution is to pick a library compatible with this one:
 
+<!-- TODO: make sure that these are all currently compatible -->
 * [Loadable Components](https://github.com/smooth-code/loadable-components/)
```

Comment:
There is a test in the examples. It's a bit complicated, as long as you can test only one library in a single point of time (or they would help each other to update).
Probably the last version of loadable components is not reload-friendly.